MParT/MultivariateExpansionWorker.h

```diff
@@ -454,6 +416,136 @@ class MultivariateExpansionWorker
 
 private:
 
+    KOKKOS_FUNCTION double GetTermVal(unsigned int termInd, const double* polyCache) const {
+        // Compute the value of this term in the expansion
+        double termVal = 1.0;
+        unsigned int end_idx = multiSet_.nzStarts(termInd+1)-1;
+        for(unsigned int i=multiSet_.nzStarts(termInd); i<end_idx; ++i)
+            termVal *= polyCache[startPos_(multiSet_.nzDims(i)) + multiSet_.nzOrders(i)];
+        
+        double lastVal = polyCache[startPos_(multiSet_.nzDims(end_idx)) + multiSet_.nzOrders(end_idx)];
+        if constexpr(!std::is_same_v<Rectifier,Identity>){
```

Comment:
Why do we need to check this here?  I wouldn't expect any performance hit if we just called `Rectifier::Evaluate`, especially if `Rectifier::Evaluate` is an inline function.